docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
+description: Learn more about searching and broadcasting available plugins for Porter
+---
+
+## Search
+
+Porter maintains a list of plugins that users can search via
+`porter plugin search [NAME]`. If no name is supplied, the full listing will be
```

Comment:
If you decide to change the help text (above), this should be updated as well to reflect whatever it is called, e.g. query or term. `If no query is supplied`.